### PR TITLE
Missing instruction for minikube when using Linux

### PR DESCRIPTION
**Pull request by gkwan-ibm**
- add instruction to start minikube
- update instruction to the openapi/ui url

### CODE REVIEW

**Review by tt-le — APPROVED** [review]
LGTM on LGDEV. Ran end to end on linux VM

File: README.adoc [hunk 1]
```diff
@@ -330,8 +338,21 @@ cart-deployment-98f4ff789-6rvfj  1/1    Running  0         17s
...
+
+[.tab_content.windows_section.mac_section]
+--
+Point your browser to the link:http://localhost:31000/openapi/ui/[^] URL.
+This URL displays the available REST endpoints.
+--
+
+[.tab_content.linux_section]
+--
+Run the `minikube ip` command to find out the host name for minikube.
```
Comment (Charlotte-Holt) [inline README.adoc:352]: Suggested update

>Run the `minikube ip` command to get the hostname for minikube.

File: README.adoc [hunk 2]
```diff
@@ -330,8 +338,21 @@ cart-deployment-98f4ff789-6rvfj  1/1    Running  0         17s
...
+Point your browser to the `http://[hostname]:31000/openapi/ui/` URL.
```
Comment (Charlotte-Holt) [inline README.adoc:353]: Suggested update

>Then, go to the `http://[hostname]:31000/openapi/ui/` URL in your browser.

**Review by Charlotte-Holt — COMMENTED** [review]
@gkwan-ibm Just two quick ID review comments. Thx!

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc]: addressed

File: README.adoc (hunk 2 above)
Comment (gkwan-ibm) [reply README.adoc]: addressed

**Review by Charlotte-Holt — APPROVED** [review]

**Pull request merged**